### ISSUE TITLE
drawTextEx not working properly

### ISSUE THREAD

**Issue by barbafh3** [Anut-py/h-raylib#6]
I tried drawing text using drawTextEx and it mostly won't work, and sometimes it shows for a frame or 2 then nothing again. Tried using this code with two different fonts I know are working from another project: 

```haskell
mainFontPath :: String
mainFontPath = "/path/to/font"

main :: IO ()
main = do
  initWindow screenWidth screenHeight "Font Test"
  setTargetFPS 75

  mainFont <- loadFont mainFontPath
  gameLoop mainFont

gameLoop :: Font -> IO ()
gameLoop mainFont = do
  beginDrawing
  clearBackground rayWhite

  drawTextEx mainFont "Testing drawTextEx" (Vector2 640.0 12.0) 20.0 1.0 black

  endDrawing

  shouldClose <- windowShouldClose
  unless shouldClose $ gameLoop mainFont
```

And the logs show the font loaded successfully

**Comment by Anut-py** [comment]
This looks like a bug with `loadFont`. The font data isn't being marshalled properly into Haskell, which corrupts the font data. I'll work on this and try to find the source.

**Comment by Anut-py** [comment]
I have fixed this in version `4.5.0.6`, please close the issue if it fixed the problem on your end.

**Comment by barbafh3** [comment]
Confirmed. Retried the example and the text showed properly.
